### ISSUE TITLE
Consider automating scan date for a11y

### ISSUE THREAD

**Issue by micahsaul** [18F/pulse#644]
HTTPS and analytics both autopopulate the scan date in the header. a11y should probably do the same.

**Comment by konklone** [comment]
More than that -- the a11y scanning process should be refactored to fit into the main pipeline. Or the main pipeline could be refactored at the same time, but, in general, all scanners should use the same data loading workflow.
